app/models/user.rb

```diff
@@ -10,6 +10,8 @@ class User < ActiveRecord::Base
   has_many :instances, through: :instance_users
   has_many :course_users, inverse_of: :user, dependent: :destroy
   has_many :courses, through: :course_users
+  has_many :group_users, inverse_of: :user, dependent: :destroy, class_name: Course::GroupUser.name
```

Comment:
So in <code>Course::Group</code> we still use <code>group_user</code> ? and in <code>User</code> we use <code>course_group_user</code> ?